Update glossary.md (#7)
nsidc/usaon-vta-docs: glossary.md

Author: Is there a difference between response objects and survey objects?

Author: We should also be consistent in using `respondent` or `expert` or differentiating between the two.

Reviewer: I've been trying to use the term `Expert` to refer to the person/role, but did use `respondent` to refer to the field on a response object that links it to the `Expert`. What do you think?

Author: I would suggest using `respondent` since it has a natural link between a `respondent` and their response. Does that make sense? As administrators, we try to make sure that the `respondents` have expertise in the things they are rating, etc. but `respondent` more accurately captures their role.

Reviewer: Are we ditching the term "Expert"? I don't see any use for it if it means the same as "Respondent"!

* `Library`: The collection of all `responses`.
* `Registry`: The collection of available `survey objects` for use in `responses`.
* `Analysis`: The process of converting many survey responses to a Sankey diagram by an

>   `Analyst` interacting with the `Library`. Surveys are searched by title, `tags`, and/or `object metadata`.

Reviewer: How would you define "object metadata"?

Author: `Object metadata` refers to the fields like `organization`, `funder`, `website` - all the fields that would be used to describe a particular `survey object`. Please feel free to update this language if there is a better way to describe it.

Reviewer: I'd prefer not to create a new term for the fields on a survey object. What do you think of this?
```suggestion
  `Analyst` interacting with the `Library`. Surveys are searched by title, `tags`, and/or other survey object fields.
```

Author: This works

> ## Registered Survey Object Registry

Reviewer: > Is there a difference between response objects and survey objects?

I would prefer to use the term  "Response" here, since these will be chosen during the response process, not the survey creation process. Would also prefer to avoid using the term "register" twice!

```suggestion
## Response Object Registry
```

Author: Sounds good

A list of available `response objects`. Drives consistency in naming across `surveys` to
support `analysis` and also to ease data entry for `Experts`, over time.

> Experts may add new `applications` `data products` and `observing systems` to the `registry`.

Reviewer: ```suggestion
Experts may add new `applications`, `data products`, and `observing systems` to the `registry`.
```

Author: Do I need to do anything to resolve/approve these changes?

Is suspect that we can just have a basic `Registered Survey Object` that has generic
fields whether it is a Data Product, Observing System, Etc. We probably need to have a
separate structure to register Societal Benefit Analysis framework.

We want to align this data structure with best practices and partner organization
structures (e.g. Polar Observing Assets working group, Arctic Data Center, Federated 
Search crew). Evenually it's possible that we could import or sync items in the registry 
from an external source.

> 
> * `Name`
> * `Type`: `Observing system`, `data product`, `application`, `SBA`
> * _TODO_: Other fields?
> 
> ## Registered Survey Object Fields
> 
> _NOTEfromHAZEL_ I think we can consolidate the `Observing System`, `Data Product`, 
> and `Application` sections below into this single list describing all survey objects, 
> except `Societal Benefit Areas`

Reviewer: ```suggestion

### Fields

```

I think this should be a subsection of the "Response object registry" section. These entities are intended to be re-usable classes of `Observing System`s, `Data Product`s, and `Application`s

  referred to as an abstract (see Arctic Data Center). 
* `Contact Name`: The name of a point of contact for the object, e.g. a data manager or program coordinator.
* `Contact Title`: The title of a point of contact for the object.
* `Contact Email`: The email address for a point of contact for the object.
* `Tags`: This is a user-defined field that admins can and will edit. It creates additional flexibility in 
  the analyis, for example allowing for a regionally- or thematically-focused `analysis`.
* `Version`: This field allows the `analyses` to reflect updates overtime. Likely
  this field will have to be open text so it can match the versioning information that the organization uses.
* `Persistent Identifier`: A standard way to refer back to the object's source, usually a ROR or DOI

`Application` objects also include:
* `Application Performance Criteria`: Text description of what the ideal performance of this data
  product looks like.

## Rated Instance of an Object

* _NOTEfromHAZEL_ I think we can consolidate many of the sections below into this single
  description of links between the `Observing System`, `Data Product`, and `Application` 

>   objects. Links to `Societal Benefit Areas` require separate definitions.

Reviewer: I like this idea!

This answers the questions - how important is a particular `Observing System`, 
`Data Product`, or `Application`and how well does it perform. It is reflected in the 

> `analysis`by the thicknessand color of the link connecting two `survey objects`. For 

Reviewer: ```suggestion
`analysis` by the thickness and color of the link connecting two `survey objects`. For 
```

`data product` but performs poorly because of persistent Arctic cloud cover and high 
latency (`gaps`. Those would be linked by a thick (high `criticality`) red (low 
`performance`) line. 

Response objects exist both as a definition in the `registry` and an instantiation with
rating(s) and other fields associated with a `response`. The following specifications
pertain to rated instances, _not_ `registry` definitions.

* `Link`: Defines which `survey objects` are connected. The rating is applied to that `link`.
  _TODO_: Research ontologies around links/provinence - 
  https://www.w3.org/TR/prov-o/#cross-reference-starting-point-terms
* `Performance rating`: 0-100 rating of the performance of the subject. Answers the question: 
  What is your satisfaction with this input? (0=No performance, 100 = perfect)
* `Criticality rating`: 0-10 rating of the criticality of an input to an output,
  e.g. criticality of an `observing system` to a `data product`. This answers the question:
  On a scale of 1-10, how much would the loss of this input impact the performance of your 
  `data product` or `application` (1 - very little impact; 10 - complete loss of performance).

> * `Rationale`: Why a `criticality` or `performance` rating was selected.

Reviewer: Should there be two rationale fields, one for criticality and one for performance?

Author: Yes, we've only had one field in the past but I think as we move towards a more standard structure that makes sense

Reviewer: - [ ] Create two `rationale` fields

* `Gaps`: If the rating is less than "ideal" what improvements are needed.
* `Variable or Attribute`: If an `observing system` or `data product` contains many

>   observable properties or variables, this allows a `respondent` to specific

Reviewer: ```suggestion
  observable properties or variables, this allows a `respondent` to specify
```

  which field they used. 

> * `Rated by`: Link to the `respondent` who provided this rating.

Reviewer: ```suggestion
* `Rated by`: Link to the `expert` who provided this rating.
```

Author: See my comment above. If you have a preference for `expert` I could be convinced.

Reviewer: No strong preference, I just think we should pick only one and get rid of the other. I don't remember where the term "expert" came from originally. I think I find "respondent" slightly more self-explanatory.

Author: Agree. Let's use "respondent" throughout.

  instead? e.g. a submitted response has `submitted_date` populated, a draft does not)

### Response objects 

> Response objects exist both as a definition in the `registry` and an instantiation with

Reviewer: I think we can get rid of everything in this section except for SBA, right?

Reviewer: 808336a
